docs/edgeql/funcops/datetime.rst

```diff
@@ -562,3 +562,27 @@ Date and Time
         {<duration>'1:20:45'}
         db> SELECT to_duration(seconds := 4845);
         {<duration>'1:20:45'}
+
+
+.. eql:function:: std::duration_to_seconds(cur: duration) -> float64
```

Comment:
First, I think we should return a decimal here.

Second, do we need this as a function?  Why don't we just enable `<decimal>` and `<float64>` casts for `std::duration`?

Reply:
Decimal cast is a bit unclear: what units should be? Python users would say seconds but javascript users would expect milliseconds.

Reply:
I was thinking about an integer number, which can only be of microseconds, but a fractional number of seconds as a `decimal` works just as well.